distribution/dashboard/internal/model/response/obtenant.go

```diff
@@ -1,12 +1,49 @@
 package response
 
-import "github.com/oceanbase/ob-operator/api/v1alpha1"
+// @Description Brief information about OBTenant
+type OBTenantBrief struct {
+	Name        string            `json:"name"`        // Name of the resource
+	Namespace   string            `json:"namespace"`   // Namespace of the resource
+	TenantName  string            `json:"tenantName"`  // Name of the tenant in the database
+	ClusterName string            `json:"clusterName"` // Name of the cluster belonging to
```

Comment:
is this ClusterName the CR name or the cluster name in OceanBase

Reply:
This is name of CR. Tenant would not feel cluster name in the OceanBase.